### ISSUE TITLE
`0.2.0` breaks config check

### ISSUE THREAD

**Issue by ndarville** [markdownlint/markdownlint#81]
Something in the update breaks the old script in [my repo](https://github.com/ndarville/style).

`0.1.0` @ Gemfile:

``` sh
$ bundle exec mdl -c markdown/.mdlrc README.md 

```

`0.2.0` @ Gemfile:

``` sh
$ bundle exec mdl -c markdown/.mdlrc README.md 
/Users/darville/.rvm/gems/ruby-2.1.0/gems/mdl-0.2.0/lib/mdl.rb:20:in `[]': no implicit conversion of Symbol into Integer (TypeError)
    from /Users/darville/.rvm/gems/ruby-2.1.0/gems/mdl-0.2.0/lib/mdl.rb:20:in `run'
    from /Users/darville/.rvm/gems/ruby-2.1.0/gems/mdl-0.2.0/bin/mdl:10:in `<top (required)>'
    from /Users/darville/.rvm/gems/ruby-2.1.0/bin/mdl:23:in `load'
    from /Users/darville/.rvm/gems/ruby-2.1.0/bin/mdl:23:in `<main>'
    from /Users/darville/.rvm/gems/ruby-2.1.0/bin/ruby_executable_hooks:15:in `eval'
    from /Users/darville/.rvm/gems/ruby-2.1.0/bin/ruby_executable_hooks:15:in `<main>’
```

Couldn’t find something in the changelog to note it, but could just be a bug in the code. :)

**Comment by mivok** [comment]
Thanks for the report. Yes, this is a bug - the config should work as is.
